rllib/connectors/common/frame_stacking.py

```diff
@@ -13,7 +13,7 @@
 
 
 @PublicAPI(stability="alpha")
-class _FrameStackingConnector(ConnectorV2):
+class _FrameStacking(ConnectorV2):
```

Comment:
Changed naming convention from `...Connector` to just `...` for simplicity and shorter (but more descriptive) names.